src/node_v8_platform-inl.h

```diff
@@ -96,28 +96,37 @@ struct V8Platform {
     // Tracing must be initialized before platform threads are created.
     platform_ = new NodePlatform(thread_pool_size, controller);
     v8::V8::InitializePlatform(platform_);
+#endif  // NODE_USE_V8_PLATFORM
   }
 
   inline void Dispose() {
+    if (platform_) {
```

Comment:
Style: `platform_ != nullptr` (no coercion to bool); also happens further down below.

Substance: I suspect it's unsound to shutdown the platform before the tracing agent (and if not now, then probably sometime in the future.)